src/backend/gl/src/lib.rs

```diff
@@ -745,3 +745,7 @@ impl gfx_core::DeviceFence<Resources> for Device {
         }
     }
 }
+
+mod gl {
+    include!(concat!(env!("OUT_DIR"), "/gl_bindings.rs"));
```

Comment:
Isn't this precisely what `gfx_gl` allowed us to avoid? If I'm not mistaken, one of the points behind that crate was that the generated bindings could be re-used without fetching the GL specs every time. @bjz - please correct me if I'm wrong here.